src/components/new-safe/create/steps/OwnerPolicyStep/useSafeSetupHints.ts

```diff
@@ -18,18 +19,26 @@ export const useSafeSetupHints = (
     }
```

Comment:
Do we need to also adjust this condition in line 13 because of the change to optional parameters? In theory it could happen now that we display something like `1/undefined policy` in case we pass a threshold but no noOwners.